doc/getting-started/rt_industry.rst

```diff
@@ -157,7 +157,7 @@ Use the pre-installed industry ACRN hypervisor
    .. note:: Ensure that ACRN is first in the boot order, or you may use ``efibootmgr -o 1`` command to move it
       to the first position. If you need to enable the serial port, run the following command before rebooting:
 
-      ``efibootmgr -c -l '\EFI\acrn\acrn.efi' -d /dev/sda -p 1 -L ACRN -u "uart=port@0x3f8"``
+      ``efibootmgr -c -l '\EFI\acrn\acrn.efi' -d /dev/sda -p 1 -L ACRN -u "uart=port@0x3f8 "``
```

Comment:
You'll need the note here too

Reply:
@gvancuts Thanks! FYI, I will also add this to another file that has this same issue (running_deb_as_serv_vm.rst).